sympy/physics/quantum/gate.py

```diff
@@ -667,7 +667,7 @@ class HadamardGate(HermitianOperator, OneQubitGate):
     sqrt(2)*|0>/2 - sqrt(2)*|1>/2
     >>> # Hadamard on bell state, applied on 2 qubits.
     >>> psi = 1/sqrt(2)*(Qubit('00')+Qubit('11'))
-    >>> qapply(HadamardGate(0)*HadamardGate(1)*psi)
+    >>> qapply(HadamardGate(0)*HadamardGate(1)*psi).expand()
     sqrt(2)*|00>/2 + sqrt(2)*|11>/2
```

Comment:
Is the `expand` necessary here? I tried this and got the same result without expand.

Reply:
You are right. This slipped through when I removed the .expand() calls after adding moderate expansion to qapply. Will be removed in the next commit.